Parse/PFPush.h

```diff
@@ -53,7 +53,7 @@ PF_TV_UNAVAILABLE PF_WATCH_UNAVAILABLE @interface PFPush : NSObject <NSCopying>
  @param channels The array of channels to set for this push.
  Each channel name must start with a letter and contain only letters, numbers, dashes, and underscores.
  */
-- (void)setChannels:(nullable NSArray PF_GENERIC(NSString *) *)channels;
+- (void)setChannels:(nullable NSArray PF_GENERIC(NSString *)*)channels;
```

Comment:
Leaving this one as well, since this style is closer to the rest of our codebase.